### PR TITLE
home/carousel

### PR DESCRIPTION
**Pull request by MarCortina**
##Summary
Screen

https://user-images.githubusercontent.com/72997924/140570528-31ff411a-06bf-41c7-880f-98fa9f697b01.mp4

Add carousel component with info images , name & description
In the services, bring the data and check and save the one with images
Module styles provisory

### CODE REVIEW

File: src/Components/Carousel/Carousel.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        pagination={{
+          clickable: true,
+        }}
+        loop={true}
+        navigation={true}
+        Autoplay={true}
+        autoplay={{
+          delay: 5000,
+        }}
+      >
```
Comment (saralavanchy) [inline src/Components/Carousel/Carousel.js:48]: the styles should come from a styles pag

File: src/Components/Carousel/Carousel.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  Autoplay,
+  Navigation,
+  Pagination,
+  Scrollbar,
+  A11y,
+  Parallax,
+} from "swiper";
+import "swiper/swiper-bundle.min.css";
+import "swiper/swiper.min.css";
+import Styles from "./Carousel.module.css";
```
Comment (saralavanchy) [inline src/Components/Carousel/Carousel.js]: the styles should be on the styles folder separated by component type: container, buttons, etc.

File: src/Components/Carousel/Carousel.module.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
+.containerCarousel {
```
Comment (saralavanchy) [inline src/Components/Carousel/Carousel.module.css]: this styles should come from Styles folder and on the styles folder should be separated by components

File: src/Services/ongInfoServices.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import axios from "axios";
```
Comment (saralavanchy) [inline src/Services/ongInfoServices.js]: this should be on slidesService, what you are returning on this endpoint is slides with images so, on slides service you should add a method to return getImagesSlides or something like that

**Review by saralavanchy — CHANGES_REQUESTED** [review]

File: src/Components/Carousel/Carousel.js (hunk 1 above)
Comment (MarCortina) [reply src/Components/Carousel/Carousel.js:48]: changes added

File: src/Components/Carousel/Carousel.js (hunk 2 above)
Comment (MarCortina) [reply src/Components/Carousel/Carousel.js]: changes added

File: src/Components/Carousel/Carousel.module.css (hunk 3 above)
Comment (MarCortina) [reply src/Components/Carousel/Carousel.module.css]: changes added

File: src/Services/ongInfoServices.js (hunk 4 above)
Comment (MarCortina) [reply src/Services/ongInfoServices.js]: changes added

**Review by saralavanchy — APPROVED** [review]

**Pull request merged**